src/fetcher.js

```diff
@@ -516,7 +516,8 @@ var Fetcher = function Fetcher (store, timeout, async) {
     } else {
       // See https://www.iana.org/assignments/link-relations/link-relations.xml
       // Alas not yet in RDF yet for each predicate
-      predicate = kb.sym(Uri.join(rel, 'http://www.iana.org/assignments/link-relations/'))
+      /// encode space in e.g. rel="shortcut icon"
+      predicate = kb.sym(Uri.join(encodeURIComponent(rel), 'http://www.iana.org/assignments/link-relations/'))
```

Comment:
1) The way things evolve in things like HTML tags and link relations and HTTP headers is always a mixture of standards first and code first.
2) yes, and IANA should by linked data principles should provide RDF at the end of the link.  This is a longstanding issue over years.  (That's why the classes derived from mime types I code to be in W3c space.
A to-do is to populate that space in fact with triples  the triples to go int it, they need t e put  in files )
3) Another thing we cna do is make a mirror of what we sould tlike they cto have, and have a list of things whih are not maintained well with pointers to where pothers mainatin a better copy -- just for core things like tis